index.bs

```diff
@@ -1756,6 +1813,8 @@ has the following [=struct/items=]:
 : <dfn for="pull-into descriptor">reader type</dfn>
 :: Either "`default`" or "`byob`", indicating what type of [=readable stream reader=] initiated this
    request
+: <dfn for="pull-into descriptor">read fully</dfn>
+:: A boolean flag indicating whether this is a {{ReadableStreamBYOBReader/readFully()}} request.
```

Comment:
Nit-pick: do we prefer enums over boolean flags? Something like "request type" which can be "default read", "byob read" or "byob read fully"?

Or perhaps we can combine this with "reader type"?

Reply:
Boolean seems fine to me. If we can combine them (i.e., if one element in the 2x2 matrix is not currently filled out) that sounds even nicer.

Reply:
A "default read-fully" request wouldn't make much sense, since there's no BYOB view that needs to be filled. The only reasonable thing this could do is keep reading until the stream becomes closed, but then `readable.arrayBuffer()` (#1019) seems like a better solution.

I'll keep it a boolean for now. 🙂